Expensify/App
# Create the Report Details Page

### Details
This PR adds the Report Details Page for default rooms and implements the ability to set notification preferences for a chatroom via a Picker.

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/161785

### Tests
1. Navigate to a default room.
2. Clicked on the header for the default room. Verify it pulls up the report details page, and that it renders correctly.
3. Change the notification preference. Verified that the Picker is updated to reflect the new notification preference, and that the API call to `Report_UpdateNotificationPreference` succeeds.
4. Click on the "Members" menu item. Verify that the members list renders correctly.

### QA Steps
Same as the above tests

### Tested On


### Screenshots


#### Web
https://user-images.githubusercontent.com/31285285/123371437-5a417180-d5b4-11eb-9136-d0e960fc5388.mp4

#### Mobile Web


#### Desktop
https://user-images.githubusercontent.com/31285285/123371868-1b5feb80-d5b5-11eb-9a51-9ccbf48a860f.mp4

#### iOS
https://user-images.githubusercontent.com/31285285/123374490-c377b380-d5b9-11eb-9d57-104f52d376a9.mp4

#### Android
https://user-images.githubusercontent.com/31285285/123375089-c7580580-d5ba-11eb-951e-6192bb38a0b2.mp4

cc @yuwenmemon
